clients/drcachesim/CMakeLists.txt

```diff
@@ -742,9 +742,10 @@ if (BUILD_TESTS)
              COMMAND tool.drcachesim.miss_analyzer_unit_test)
 
     add_executable(tool.drcachesim.invariant_checker_test tests/invariant_checker_test.cpp)
+    configure_DynamoRIO_standalone(tool.drcachesim.invariant_checker_test)
```

Comment:
So this is because _standalone invokes _DR_set_compile_flags which for non-clients restores the ORIG_ versions at the source file level.

I wonder if what we should do is add a new configure_original_flags(target) which does this for all our other targets in this file which are neither clients nor use DR standalone, and then get rid of add_win32_flags() which looks like it just adds a couple of flags back when it should use the ORIG_ values.  Add a `TODO i#6040` comment by add_win32_flags?